scripts/docs-generation/3rd-party/services.json

```diff
@@ -52,5 +52,11 @@
     "title": "LambdaTest",
     "githubUrl": "https://github.com/LambdaTest/wdio-lambdatest-service",
     "npmUrl": "https://www.npmjs.com/package/wdio-lambdatest-service"
+  },
+  {
+    "packageName": "wdio-image-comparison-service",
+    "title": "Visual Regression Testing",
+    "githubUrl": "https://github.com/wswebcreation/wdio-image-comparison-service",
```

Comment:
Should this be added if the the tool used for comparison is still being developed? https://github.com/wswebcreation/webdriver-image-comparison

Reply:
@emilbryggare 

The webdriver-image-comparison is still being developed, it's the core of the comparison module, but can't be used as a standalone module together with WebdriverIO